src/startupscriptgenerator/src/common/parseExtensibleConfig.go

```diff
@@ -0,0 +1,49 @@
+// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
+
+package common
+
+import (
+    "os"
+    "strings"
+    "gopkg.in/yaml.v3"
+)
+
+type EnvironmentVariable struct {
+    Name  string `yaml:"name"`
+    Value string `yaml:"value"`
+}
+
+type ExtensibleConfigFile struct {
+    BaseOs               string                `yaml:"base-os"`
+    Platform             string                `yaml:"platform"`
+    PlatformVersion      string                `yaml:"platform-version"`
```

Comment:
Just wanted to confirm that at this point, we don't expect for platform, platform-version, or base-os to be respected or required here?

Reply:
Not at this current moment, but it _could_ be something that we use in the future for the runtime script, so I'm just keeping them here for now. I don't have any gripes with removing them if you feel that it would be better for keeping the file clean though.

Reply:
Nope, I'm all good with keeping them here for now